client/components/main/About.jsx

```diff
@@ -64,30 +65,70 @@ const About = () => {
               the expansive amount of data associated with these 311
               requests is available online. The mayor has encouraged
               us to create apps with this data, and that&apos;s where
-              this project comes in.
+              this project comes&nbsp;in.
             </p>
             <h2 style={{ color: '#1D6996', textAlign: 'center' }}>Partners</h2>
-            <p>
+            <p style={{ paddingBottom: '1.625rem' }}>
               To empower local residents and Neighborhood Councils to
               make informed decisions about how to improve their communities
               using an easy-to-use application, EmpowerLA partnered with Hack
               For LA  to create the 311 Data project. The 311 Data project makes
               navigating the wealth of 311 data easier using an open source
-              application built and maintained by volunteers throughout our community.
+              application built and maintained by volunteers throughout our&nbsp;community.
             </p>
           </Grid>
         </Grid>
-        <Grid container>
-          <Grid item sm={12} md={4} className={classes.gridItem}>
+        <Grid container style={{ paddingBottom: '6rem' }} className={classes.gridContain}>
+          <Grid item xs={12} md={4}>
             <img src={empowerLaLogo} width={362} alt="Empower LA" />
           </Grid>
-          <Grid item sm={12} md={4} className={classes.gridItem}>
+          <Grid item xs={12} md={4}>
             <img src={hackForLaLogo} width={97} alt="Hack for LA" />
           </Grid>
-          <Grid item sm={12} md={4} className={classes.gridItem}>
+          <Grid item xs={12} md={4}>
             <img src={codeForAmericaLogo} width={202} alt="Code for America" />
           </Grid>
         </Grid>
+        <h2 style={{ color: '#1D6996', textAlign: 'center' }}>How it works</h2>
+
+        <Grid container style={{ paddingBottom: '4rem' }} className={classes.gridContain}>
+          <Grid item sm={12} md={2} style={{ textAlign: 'center' }}>
```

Comment:
all instances of `style={{ textAlign: 'center' }}` can be replaced with `align="center"`

Reply:
Same. good call.